src/main/play-doc/api/BundleAPI.md

```diff
@@ -98,7 +242,7 @@ Either the service has not been started, or the service requested cannot be foun
 
 Other status codes should also be treated as a failure.
 
-## Receive hosts events
+## Receive hosts events (deprecated - use the /v2 path instead)
```

Comment:
Can we do more specific about the replacement and add a link to it, e.g.

## Receive hosts events (deprecated - use the [/v2](#Receive-hosts-events) endpoint instead)

The link consists of `[/v2](#Receive-hosts-events)`

Same for the other deprecated endpoints.